src/dmd/tocsym.d

```diff
@@ -480,7 +480,7 @@ Symbol *toImport(Symbol *sym)
     }
     else if (sym.Stype.Tmangle == mTYman_d)
```

Comment:
The expression in the `if` branch looks superfluous since the fallback does the same line 483. Hence the style check failure.

Reply:
Sorry, bad comment. I referred to the test `if (sym.Stype.Tmangle == mTYman_d)`. It's not necessary since the final unconditional `else` does the same in its block.

Reply:
I left it in to keep the D mangling version explicit, but it's probably not too important. Removed now.

Reply:
Yes I see, thanks. My only concern was to have all the check green.